react/components/admin/institutional/README.md

```diff
@@ -0,0 +1,28 @@
+# VTEX Content Pages
+
+## Description
+
+This is just an easier way to create content pages in your store. It provides an editor for you to create new routes and their content in a single form. You can edit the content using Storefront later as well.
+
+:loudspeaker: **Disclaimer:** Don't fork this project; use, contribute, or open issue with your feature request
+
+## Usage
+
+You must add `store.content` to your `blocks.json` just like the example below:
+
+```json
+  "store.content": {
+    "children": ["flex-layout.row#content-body"]
+  },
+  "flex-layout.row#content-body": {
+    "children": ["rich-text"]
+  }
+```
+
+:loudspeaker: **Disclaimer** Content pages won't be available if this block don't be added to your `blocks.json`.
```

Comment:
:pick: I don't think this disclaimer is necessary, since we're already saying that "you **must** add `store.content` to your `blocks.json`".

Reply:
Fair enough, gonna remove it